python/pyspark/sql/tests/test_context.py

```diff
@@ -66,7 +66,7 @@ def test_save_and_load_table(self):
         tmpPath = tempfile.mkdtemp()
         shutil.rmtree(tmpPath)
         df.write.saveAsTable("savedJsonTable", "json", "append", path=tmpPath)
-        actual = self.spark.createExternalTable("externalJsonTable", tmpPath, "json")
```

Comment:
Those seems can be reverted back since the tests target HiveContext.

Reply:
IIRC this failed, because HiveContext just inherits from SQLContext.

Oh, well, this is another good point: HiveContext is deprecated in 2.0! It should just be removed, no?

Reply:
Hm, wait this fails after the change, you have a point:
```
======================================================================
ERROR: test_save_and_load_table (pyspark.sql.tests.test_context.HiveContextSQLTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/workspace/SparkPullRequestBuilder/python/pyspark/sql/tests/test_context.py", line 69, in test_save_and_load_table
    actual = self.spark.createTable("externalJsonTable", tmpPath, "json")
AttributeError: 'HiveContext' object has no attribute 'createTable'
```
Also solved by removing it